const asyncHandler = (fn) => (req, res, next) => {
    try {
      return Promise.resolve(fn(req, res, next)).catch(next);
    } catch (err) {
      next(err);
    }
  };
  
  export default asyncHandler;
  